packages/top-app-bar/README.md

```diff
@@ -32,7 +26,7 @@ prominent | n/a | Enables prominent variant.
 
 ### Navigation Icon
 
-The navigation icon can be a `<a>`, `<i>`, `<svg>`, `<image>`, `<span>`, etc. If you decide to pass a non-native `React.Component`, you need to append the prop `className` to the className prop on the parent element. As an example please see the [MaterialIcon Component](../material-icon/index.js).
+The navigation icon can be a `<a>`, `<i>`, `<svg>`, `<image>`, `<span>`, etc.
 
 ```js
 import MyIcon from './myIcon';
```

Comment:
Instead of having this code snippet which shows how to add className, just link to the docs/guidelines.md